content/agent/kubernetes/helm.md

```diff
@@ -111,39 +111,52 @@ To install the chart with the release name `<RELEASE_NAME>`, retrieve your Datad
 helm install --name <RELEASE_NAME> --set datadog.apiKey=<DATADOG_API_KEY> stable/datadog
 ```
 
-Refer to the section below for a list of configurable parameters.
-
 This chart adds the Datadog Agent to all nodes in your cluster via a DaemonSet. It also optionally deploys the [kube-state-metrics chart][5] and uses it as an additional source of metrics about the cluster. A few minutes after installation, Datadog begins to report hosts and metrics.
 
-### Configuration
+**Note**: For a full list of the Datadog chart's configurable parameters and their default values, refer to the [Datadog Helm repository README][6]. 
+
+## Configuring the Datadog Helm chart
 
-For a full list of the Datadog chart's configurable parameters and their default values, refer to the [Datadog Helm repository readme][6].
+As a best practice, a YAML file that specifies the values for the chart parameters should be provided to configure the chart: 
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example:
+1.  **Copy the default [`datadog-values.yaml`][7] value file.**
+2.  Set the `apiKey` parameter with your [Datadog API key][4].
+3. Upgrade the Datadog Helm chart with the new `datadog-values.yaml` file:
 
 ```bash
-helm install --name <RELEASE_NAME> \
-  --set datadog.apiKey=<DATADOG_API_KEY>,datadog.logLevel=DEBUG \
-  stable/datadog
+helm upgrade -f datadog-values.yaml <RELEASE_NAME> stable/datadog --recreate-pods
 ```
 
-Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example:
+### Enable Log Collection
 
-```bash
-helm install --name <RELEASE_NAME> -f my-values.yaml stable/datadog
-```
+Update your [datadog-values.yaml][7] file with the following log collection configuration, then upgrade your Datadog Helm chart:
 
-You can copy and customize the default [values.yaml][7].
+```
+datadog:
+  (...)
+ logsEnabled: true
+ logsConfigContainerCollectAll: true
+```
 
-## Upgrading the chart
+### Enabling Process Collection
 
-Use the `helm upgrade` command to upgrade to a new version of the chart, or to change the configuration of your release. 
+Update your [datadog-values.yaml][7] file with the process collection configuration, then upgrade your Datadog Helm chart:
 
-```bash
-helm upgrade -f my-values.yaml <RELEASE_NAME> stable/datadog --recreate-pods
+```
+datadog:
+  (...)
+  processAgentEnabled: true
+  volumes:
```

Comment:
wondering whether we should do the same as for logs, i.e. do we really need to have this in the `values.yaml` ? I think we could hardcode those values directly in the deamonset ? cc @xvello

Reply:
Live processes work without these files, and will show uids instead of usernames, but indeed that'd be good to just add the volumes automatically when `processAgentEnabled` is true

Reply:
Actually, it's already the case, see https://github.com/helm/charts/commit/fa9cf124fa7f0ad91754c37920de3bfdc850da87

This means only setting processAgentEnabled is enough

Reply:
ping @mikezvi for tracking